.github/ISSUE_TEMPLATE/Standard-Bug-Issue-Template.md

```diff
@@ -27,31 +27,42 @@ labels: Bug
 ## Notes
 < bulleted list of special notes or suggestions to the development team on any information pertinent to the bug. > 
 
-## Definition of Ready
-
+## Ready for Ready to Work
 - [ ] The Issue maintains standard format for Issues and meets INVEST
-- [ ] Story Defined and written appropriately
-- [ ] Steps to Recreate supplied and sufficiently detailed
-- [ ] Expected and Actual Behavior supplied
-- [ ] Business Value supplied
+- [ ] Story and Business Value complete
+- [ ] Steps to Recreate, Expected Behavior, and Actual Behavior complete
 - [ ] The person who will Approve story identified
 - [ ] Dependencies Identified
-- [ ] User acceptance test cases are created/modifed
-- [ ] Bug Issue linked to Feature Issue and/or Epic containing the functionality (optional if Epic/Feature exist)
-
-## Definition of Done
-
-- [ ] Proposed solution accepted (optional and used if the solution changes anything other than the Actual Behavior to Expected)
-- [ ] Issue accepted by PO
-- [ ] Issue is scored
-- [ ] Peer Reviews complete	
-- [ ] Unit and regression tests are Passed
-- [ ] Manual Testing Complete and issues resolved
-- [ ] Automated tests created and run successfully
-- [ ] All blockers addressed
+
+## Ready for Review
+- [ ] Proposed solution defined and accepted by Approver
+- [ ] Code implemented and built
+- [ ] Testing implemented and passed
+
+## Ready for Pre-Production QA
+- [ ] Peer Reviews complete with all changes tested
+- [ ] All features implemented
```

Comment:
With this being a bug, should the wording be around the word `feature`? Shouldn't it be more along the lines of `features are functioning properly` or something like that?

Reply:
swing and a miss on my part. will adjust accordingly